sdk/src/client/secret/mod.rs

```diff
@@ -78,9 +81,16 @@ pub trait SecretManage: Send + Sync {
         options: impl Into<Option<GenerateAddressOptions>> + Send,
     ) -> Result<Vec<EvmAddress>, Self::Error>;
 
-    /// Signs `msg` using the given [`Chain`].
+    /// Signs ed25519 key using the given [`Chain`].
```

Comment:
?

Reply:
?

Reply:
That's a bit weird? We're not signing a key but a message?

Reply:
Oh, gotcha